docs/user-guide/nodes.rst

```diff
@@ -425,31 +418,47 @@ On the Pulp server to be used as the child node:
 
   $ sudo yum install pulp-nodes-child pulp-nodes-consumer-extensions pulp-agent
 
-2. Edit ``/etc/pulp/nodes.conf`` and set the parent OAuth *key* and *secret* to match values found in
+2. Edit ``/etc/pulp/server.conf`` on the child node to configure OAuth to match the values found in
```

Comment:
This change is incorrect.  The matching `[parent_oauth]` settings are in `/etc/pulp/nodes.conf` not `server.conf`.